src/screens/MainScreen/index.js

```diff
@@ -81,7 +92,7 @@ const MainScreen = () => {
         showUserLocation
         followUserLocation>
         {topics &&
-          targets.map(({ target: { id, lat, lng, topicId } }, _, targets) => (
+          targets.map(({ target: { id, lat, lng, topicId } }, _undefined, targets) => (
```

Comment:
Targets is already defined. Is there any reason why you are getting it from `map`?

```suggestion
          targets.map(({ target: { id, lat, lng, topicId } }) => (
```

Reply:
Agree